content/tutorials/native-aa-multisig/10.index.md

```diff
@@ -964,8 +964,8 @@ export default async function (hre: HardhatRuntimeEnvironment) {
 
 ::callout{icon="i-heroicons-light-bulb"}
 ZKsync has different address derivation rules from Ethereum.
-Always use the [`createAddress`](https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#createaddress)
-and [`create2Address`](https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#create2address) utility functions of the `zksync-ethers` SDK.
+Always use the [`createAddress`](https://sdk.zksync.io/js/ethers/api/v6/utilities#createaddress)
+and [`create2Address`](https://sdk.zksync.io/js/ethers/api/v6/utilities#create2address) utility functions of the `zksync-ethers` SDK.
```

Comment:
https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#createaddress 
https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#create2address

Reply:
oh nvm these are updated in sdk pr?

Reply:
Yep, they're getting updated in the SDK PR. I'll merge that first before doing this and the ZKsync Docs one.